### DISCUSSION TITLE
Harp Clock Synchroniser

### DISCUSSION THREAD

**Discussion by EleonoraAmbrad** [bonsai-rx/bonsai#1874] [Q&A]
Hi all, we are trying to synchronise the timestamps of 3 harp devices using a clock synchroniser (https://github.com/harp-tech/device.clocksynchronizer). In terms of hardware, I have connected the harp devices to the clock outputs of the device using the jacks, and the device to the PC using the USB. The synchroniser device is also blinking, which is reassuring. My question is: in terms of software, what else needs to be implemented? I found a few "clock synchroniser" nodes on bonsai, belonging to different packages, but I am not sure how to use them. thanks in advance.

**Comment by bruno-f-cruz** [comment]
You shouldn't need any software to ensure synchronization. Synchronization is taken care of at the level of hardware/firmware.

**Reply by EleonoraAmbrad** [reply]
ok, thanks for the answer!
